### DISCUSSION TITLE
Podman Package not found

### DISCUSSION THREAD

**Discussion by divyesh98** [containers/podman#22973] [Q&A]
Hi when trying to install podman, I always get the error - package not found.  

Here are the commands I am using

`sudo apt-get update
sudo apt upgrade
sudo apt-get install -y podman`

I also tried searching for the package using

'apt search podman'

and could not find the package.

Am I missing something here?

Just to put out, I am using github codespaces,and hence network restrictions are not an issue.

**Comment by afbjorklund** [comment]
What distribution are you using? The `podman` package is available for Ubuntu 20.10 and later versions...

https://packages.ubuntu.com/podman
